File: Java/Android/OpenGLES/source/core/src/loon/foundation/NSMutableDictionary.java
package loon.foundation;

public class NSMutableDictionary extends NSDictionary {

	public static NSMutableDictionary dictionary() {
		return new NSMutableDictionary();
	}

	public void setObject(NSObject k, NSObject v) {
		_dict.put(k, v);
	}

	public void removeObject(NSObject k) {
		_dict.remove(k);
	}
}

File: C#/WindowsPhone/LGame-XNA-lib/Loon.Foundation/NSMutableDictionary.cs
namespace Loon.Foundation {

	public class NSMutableDictionary : NSDictionary {
	
		public static NSMutableDictionary Dictionary() {
			return new NSMutableDictionary();
		}
	
		public void SetObject(NSObject k, NSObject v) {
			_dict.Put(k, v);
		}
	
		public void RemoveObject(NSObject k) {
			_dict.Remove(k);
		}
	}
}
